### DISCUSSION TITLE
Update to front page info

### DISCUSSION THREAD

**Discussion by micarex** [lewisbenge/hass-amber-electric#32] [General]
@lewisbenge the info provided on the [home page](https://github.com/lewisbenge/hass-amber-electric) for this integration still relates to the original integration you wrote which requires changes to the configuration.yaml and is based on postcode.

Wondering if it should be updated to mention there is also an updated integration from @madpilot from Amber that provides more accurate values and is specific to an address?  It also requires an API token from https://app.amber.com.au/developers/ for it to work.  And lastly it is installed via a HASS integration.

**Comment by madpilot** [comment]
Probably also worth us moving this convo to the other repo.

**Reply by micarex** [reply]
Oops.  Yes I posted it in the wrong one.  Will close and move it.
